lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,33 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // Check to see if the response if from Rogue.
+      if (Drupal.settings.dsRogue.poweringGallery) {
```

Comment:
@DFurnes still need this block of code again to pull out page number to proxy via Phoenix's `/reportbacks` endpoint and then send the page number in the request to Rogue.